roles/fio_distributed/templates/client.yaml

```diff
@@ -36,25 +36,27 @@ spec:
 {% endif %}
         command: ["/bin/sh", "-c"]
         args:
-          - "cat /tmp/host/hosts;
-{% if workload_args.prefill is defined and workload_args.prefill is sameas true %}
-             echo ***************Prefill*****************;
-             cat /tmp/fio/fiojob-prefill;
-             mkdir -p /tmp/fiod-{{ uuid }}/fiojob-prefill;
-             fio --client=/tmp/host/hosts /tmp/fio/fiojob-prefill --output-format=json;
+          - cat /tmp/host/hosts;
```

Comment:
@mulbc after testing this a few times this change does not work when adding each item to its own -'d line as it stands. It would only run correctly when listed out like before, in a "" list. If this is something you want to keep working on that's fine however it does not pass ci like this.

Reply:
Got it - I'll look into this when I have some time and see how I can fix this....